codejail/jail_code.py

```diff
@@ -290,6 +292,11 @@ def jail_code(command, code=None, files=None, extra_files=None, argv=None,
 
         # Determine effective resource limits.
         effective_limits = get_effective_limits(limit_overrides_context)
+        log.info(
+            "Preparing to execute jailed code %s with resource limits: %r",
+            jail_code,
+            effective_limits,
+        )
```

Comment:
This is the only functionality change. All the others are fixing new violations from the pylint upgrade.